index.html

```diff
@@ -5,9 +5,43 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Grocery Time-Saver</title>
 		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.2/css/bulma.min.css" />
+
+		<!-- Added Style Tag to center "Let's Get Started" button in the Modal here: -->
+		<style>
+			.modal-footer {
+				display: flex;
+				justify-content: center; 
+			}
+		</style>
+
+		<!-- Added Style Tag(s) to animate waving hand emoji in the Modal here: -->
+		<style>
+			.wave {
+				animation-name: wave-animation;
+				animation-duration: 2.5s;
+				animation-iteration-count: infinite;
+				transform-origin: 70% 70%;
+				display: inline-block;
+			}
+		</style>
+
+		<style>
+			@keyframes wave-animation {
+				0% { transform: rotate( 0.0deg ) }
+				20% { transform: rotate(-8.0deg) }
+  				30% { transform: rotate(14.0deg) }
+   				40% { transform: rotate(-4.0deg) }
+  				50% { transform: rotate(10.0deg) }
+  				60% { transform: rotate( 0.0deg) }
+		
+			}
+		</style>
 	</head>
 
-	<body>
```

Comment:
Good class addition here! I'm curious as to why my page appears centered without having the class in my html code, but I think I'll understand better when Casey adds his code in.

Reply:
Thanks! I also noticed the default content on the web page was centered, however when I ran my syntax I spotted where the content in the ul wasn't in the center. I also centered the modal + content within the modal as well (which was only seen on my end bc I hadn't merged yet)